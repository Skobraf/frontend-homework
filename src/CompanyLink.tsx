import { Company } from './types'

const CompanyLink = ({ name }: Company) => (
  <div>{name}</div>
)

export default CompanyLink
