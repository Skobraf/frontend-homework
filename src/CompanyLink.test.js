import { shallow } from "enzyme";
import CompanyLink from "./CompanyLink";

describe("<CompanyLink />", () => {
  const singleCompany = [{ name: "first company" }];
  it("renders link", () => {
    expect(
      shallow(
        <CompanyLink
          id={1}
          name="Dummy company"
          singleCompany={singleCompany}
        />
      )
    ).toMatchSnapshot();
  });
});
