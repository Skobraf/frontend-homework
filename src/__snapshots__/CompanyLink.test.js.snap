// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CompanyLink /> renders link 1`] = `
<div>
  Dummy company
</div>
`;
